### ISSUE TITLE
Unable to mute hashtags/create filters

### ISSUE THREAD

**Issue by chris-y** [jointakahe/takahe#601]
Unable to mute a hashtag from Tusky.
The error just states "error muting hashtags"
I get the same problem creating a filter manually, but I think this might be what Tusky does internally for muting a hashtag anyway?

**Comment by andrewgodwin** [comment]
Yes, Takahē does not yet support server-side filtering, so this will not work until we implement it. I was hoping to do it at the same time as we do a plugin API for more advanced incoming post filtering, but we'll see.
